base_delivery_carrier_files/wizards/generate_carrier_files.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Copyright 2012 Camptocamp SA
+# Author: Guewen Baconnier
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import Warning
+
+
+class DeliveryCarrierFileGenerate(models.TransientModel):
+
+    _name = 'delivery.carrier.file.generate'
```

Comment:
Put a _description (see runbot errors)

Reply:
Done.
I had errors on reading runbot errors. But at least I could read them and solve all the problems.
Thank you for your attention.